src/sass/components/_header.scss

```diff
@@ -224,3 +254,60 @@ header {
     }
   }
 }
+
+@media (min-width: 767px) and (max-width: 1200px) {
+
+  .menu-bar {
+    height: 105px;
+    .row {
+      flex-direction: column-reverse;
+    }
+  }
+
+  .search-form-sm {
+    display: flex;
+    justify-content: center;
+    padding: 10px 0;
+  }
+
+  .menu {
+    justify-content: center;
+  }
+}
+
+@media (max-width: 767px) {
+
+  .nav-hamburger {
+    display: block !important;
```

Comment:
Using `!important` is a very bad practice. Do you have any idea to accomplish this task in another way?

This comment also applies to `!important`s below.

Reply:
I just did that in other way, think the way that should be from the start.